sql/semisync_master.cc

```diff
@@ -1499,6 +1469,16 @@ void Repl_semi_sync_master::await_all_slave_replies(const char *msg)
   DBUG_VOID_RETURN;
 }
 
+void Repl_semi_sync_master::clear_acked_tranx_nodes()
+{
+  DBUG_ENTER("refresh_wait_for_slave_count");
```

Comment:
This DBUG_ENTER function name isn't consistent with the real function name (and
note my above comments about changing the function name too).